BEASTIE/annotate_LD_new.R

```diff
@@ -4,7 +4,7 @@ suppressMessages(library(data.table))
 suppressMessages(library(foreach))
 suppressMessages(library("dplyr"))
 library("LDlinkR")
-source("Get_LD.R")
+source("./BEASTIE/Get_LD.R")
```

Comment:
You're going to have a merge conflict here, please resolve in favor of the other code.